packages/cloudflare-workers-bindings-extension/src/extension.ts

```diff
@@ -32,6 +33,13 @@ export async function activate(
 		() => bindingsProvider.refresh()
 	);
 
+	// Register the add bindings command
+	vscode.commands.registerCommand(
```

Comment:
We should add this to the context subscription list below so it's cleaned up properly